naplib/preprocessing/filter_hilbert.py

```diff
@@ -161,23 +160,30 @@ def filterbank_hilbert(x, fs, Wn=[1,150], n_jobs=-1):
     
```

Comment:
Unless you add a parameter like ``return_average`` and set it to False by default, the above Examples section is now broken

Reply:
Separated this functionality into two functions now: `filter_hilbert` and `filterbank_hilbert`